packages/kbn-kibana-json-schema/src/kibana_json_schema.ts

```diff
@@ -0,0 +1,145 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import type { JSONSchema } from 'json-schema-typed';
+import dedent from 'dedent';
+
+export const KibanaJsonSchema: JSONSchema = {
+  type: 'object',
+  required: ['id', 'version', 'owner'],
+  properties: {
+    id: {
+      description: dedent`
+        Identifier of the plugin. Must be a string in camelCase. Part of a plugin
+        public contract. Other plugins leverage it to access plugin API, navigate
+        to the plugin, etc.
+      `,
+      type: 'string',
+      pattern: '^[a-z]{1}([a-zA-Z0-9]{1,})$',
+    },
+    version: {
+      description: 'Version of the plugin.',
+      type: 'string',
+      pattern: '^(kibana|v?\\d+(\\.\\d+){0,2})$',
+    },
+    kibanaVersion: {
+      description: dedent`
+        The version of Kibana the plugin is compatible with, defaults to "version".
```

Comment:
```suggestion
        The version of Kibana the plugin is compatible with, defaults to "kibana".
```

Reply:
I think the "version" mention, which I copied from https://github.com/elastic/kibana/blob/main/src/core/server/plugins/types.ts#L176:L176, is referring to the version field.

Reply:
Clarified in [`a7006a3` (#130625)](https://github.com/elastic/kibana/pull/130625/commits/a7006a3f89eff9d06eb56892aa33cd22caf22ac6)